samples/cellular/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -367,6 +368,171 @@ To monitor the GNSS module (for instance, to see whether A-GPS or P-GPS assistan
+Make sure that the ``-cn`` argument is set to your chosen device ID.
+The ``-f cred_`` argument ensures all outputs from ``create_device_credentials.py`` are prefixed with ``cred_``.
+The ``-dv 2000`` specifies that the certificate should remain valid for 2000 days.
+Rember to populate the ``-c``, ``-st``, ``-l``, ``-o``, and ``-ou`` arguments with your own values.
+
+If this succeeds, you should now have an additional three files:
+
+  * ``cred_my_wifi_dev_crt.pem``
+  * ``cred_my_wifi_dev_pub.pem``
+  * ``cred_my_wifi_dev_prv.pem``
+
+These three files can only be used by a single device.
+The filenames automatically include your chosen device ID.
+If an attacker obtains the private key (``cred_my_wifi_dev_prv.pem``), they will be able to impersonate your device.
+Since the :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` does not protect this private key from extraction, it should not be used in production!
+
+Next, create a file named ``my_wifi_dev_cred.csv`` and give it the following contents:
+
+.. code-block:: csv
+
+   XXXX,,,,"YYYYYYYY"
+
+Where XXXX is replaced by your chosen device ID, and YYYYYYYY is replaced by the exact contents of ``cred_my_wifi_dev_crt.pem``.
+
+For example:
+
+.. code-block:: csv
+
+   my_wifi_dev,,,,"-----BEGIN CERTIFICATE-----
+   sCC8AtbNQhzbp4y01FEzXaf5Fko3Qdq0o5LbuNpVA7S6AKAkjt17QzKJAiGWHakh
+   RnwzoA2dF4wR0rMP5vR6dqBblaGAA5hN7GE2vPBHTDNZGJ6tZ9dnO6446dg9gGds
+   eeadE1HdVnUj8nb+7CGm39vJ4fuNk9vogH0nMdxjCnXAinoOMRx8EklQsR747+Gz
+   sxcdVYuNEb/E2vWBHTDNZGJ6tZC1JC9d6/RC3Vb1JC4tWnK9mk/Jw984ZuYugpMc
+   1t9umoGFYCz0nMdxjCnXAbnoOMC5A0RxcWPzxfC5A0RH+j+mwoMxwhgfFY4EhVxp
+   oCC8labNQhzRC3Vc1JC4tWnK9mpVA7k/o5LbuNpVA7S6AKAkjt17QzKJAiGWHakh
+   RXwcoAndF4wPzxfC5A0RHponmwBHTDoM7GE2vPBHTDNZGJ6tZ9dnO6446dg9gGds
+   eefdE1HcVnULbuNpVA7S6AKAkjxjCnt1gH0nMdxjCnXAinoOMRx8EklQsR747+Fz
+   srm/VYaNEb/E2vPBHTDNZGJ6tZc1JC9d6/RC3Vc1JC4tWnK9mk/Jr984ZuYugpMc
+   nt9uZTGFYCzZD0FFAA5NAC4i1PARStFycWPzxfC5A0RqodhswoMxwhgfFY4EhVx=
+   -----END CERTIFICATE-----
+   "
+
+Do not attempt to use this example verbatim, it has been filled with dummy data.
+
+Next, navigate to the `Provision Devices <nRF Cloud Portal Provision Devices_>`_ page of the nRF Cloud portal, drag and drop ``my_wifi_dev_cred.csv``, and press ``Upload and Provision``.
+You should see a message stating that the file was uploaded successfully, and a device with your chosen device ID should appear in the `Devices <nRF Cloud Portal Devices_>`_ page.
+
+Next, download `Amazon Root CA 1`_ and save it as ``AmazonRootCA1.pem``.
+See `CA certificates for server authentication in AWS IoT Core`_ for more details.
+Do not confuse this CA cert with your self-managed CA cert.
+
+Next, wrap ``cred_my_wifi_dev_crt.pem``, ``cred_my_wifi_dev_prv.pem``, and ``AmazonRootCA1.pem`` in quotes, and save them as ``client-cert.pem``, ``private-key.pem`` and ``ca-cert.pem`` respectively.
```

Comment:
maybe lead in with something like "in order to use these files in your firmware application, they must be formatted/named blah blah..."

Reply:
As someone not familiar with certificates, I have no idea how to parse "wrap <file> in quotes". Does it refer to the contents? This also seems like two different steps. One for the quotation marks, one for the renames.

Reply:
> Does it refer to the contents? 

Yes, indeed, I will adjust the sentence to specify that

> This also seems like two different steps. One for the quotation marks, one for the renames.

It is a single step, since the command lines process the text in memory and save the result to a new file name. But I can phrase it as `save the result` rather than `rename`